### PR TITLE
Update docs to v1.0.0

### PR DESCRIPTION
**Pull request by wjt**
Plus some unmerged stuff from
https://github.com/flatpak/flatpak/pull/1995.

### CODE REVIEW

File: reference/html/flatpak-Error-codes.html [hunk 1]
```diff
@@ -38,14 +38,24 @@ <h2><span class="refentrytitle"><a name="flatpak-Error-codes.top_of_page"></a>Er
 <col width="150px" class="functions_return">
 <col class="functions_name">
 </colgroup>
-<tbody><tr>
+<tbody>
+<tr>
 <td class="function_type">
-<span class="returnvalue">GQuark</span>
+<a href="/usr/share/gtk-doc/html/glib/glib-Quarks.html#GQuark"><span class="returnvalue">GQuark</span></a>
```
Comment (wjt) [inline reference/html/flatpak-Error-codes.html]: Hmm I guess links like these should be suppressed or made to point to the GNOME docs website.

**Comment by alexlarsson** [comment]
Isn't this supposed to be updated in the flatpak-docs git repo? Does anything still use the old gh-pages stuff?

**Comment by wjt** [comment]
Not sure but flatpaks.github.io/... has highest Google juice for
the API docs.

On Mon, 27 Aug 2018, at 11:34, Alexander Larsson wrote:
> Isn't this supposed to be updated in the flatpak-docs git repo? Does
> anything still use the old gh-pages stuff?> — You are receiving this because you authored the thread. Reply to
> this email directly, view it on GitHub[1], or mute the thread[2].
Links:

  1. https://github.com/flatpak/flatpak/pull/1999#issuecomment-416185446

**Comment by alexlarsson** [comment]
I think we should update flatpak-docs too though.

**Comment by wjt** [comment]
The C API documentation doesn't seem to be anywhere in https://github.com/flatpak/flatpak-docs or visible anywhere on https://flatpak.readthedocs.io/en/latest/ FWIW

**Pull request merged**